lib/src/DateRangePicker/DateRangePickerViewDesktop.tsx

```diff
@@ -25,6 +25,7 @@ export interface ExportedDesktopDateRangeCalendarProps {
    * @default 2
    */
   calendars?: 1 | 2 | 3;
+  renderCalendarInfo?: React.ReactNode;
```

Comment:
Will be better to have a function that will take the currentMonth as an argument.